awx/main/management/commands/inventory_import.py

```diff
@@ -97,12 +91,6 @@ def get_base_args(self):
                 logger.debug('Using system install of ansible-inventory CLI: {}'.format(potential_path))
                 return [potential_path, '-i', self.source]
 
-        # Stopgap solution for group_vars, do not use backported module for official
-        # vendored cloud modules or custom scripts TODO: remove after Ansible 2.3 deprecation
-        if self.is_vendored_source or self.is_custom:
-            self.method = 'inventory script invocation'
-            return [self.source]
```

Comment:
@wenottingham Can we safely assume that 2.3 is no longer supported for current development?

Reply:
... as long as custom venvs aren't used for inventory import, sure!

Reply:
The docs don't seem to mention this

https://github.com/ansible/awx/blob/devel/docs/custom_virtualenvs.md

There has been a request for that... but I don't see any scenario where it's worth it to keep the backport for this reason.

As a middle ground, we could simply test the Ansible version inside of imports and raise a descriptive error that they need Ansible 2.4 or higher.

Reply:
From https://docs.ansible.com/ansible-tower/latest/html/installandreference/requirements_refguide.html:

> Ansible version 2.2 (at minimum) required to run Ansible Tower versions 3.2 and later

So I don't think we'll be getting away with removing 2.3 support in the _very_ new future.  It's also worth noting that Ansible 2.4.0 is _only_ just over a year old.  I think we should continue to hold off on this one for awhile longer.